source/people/main.rst

```diff
@@ -89,7 +89,8 @@ Undergraduate Students & Interns
 +------------------------------------+
 |                                    |
 |                                    |
-|  :doc:`Ellis Wilson <ejwilson3>`   |
+| * :doc:`Ellis Wilson <ejwilson3>`  |
+| * :doc:`Alex Swenson <aaswenson>`  |
```

Comment:
Rather than share a box with Ellis, your name and photo should be in a separate adjacent box similar to the grad students section.